testsuite/run_cim_operations.py

```diff
@@ -331,7 +320,7 @@ def assertInstancesValid(self, instances, includes_path=True,
 
             if includes_path:
                 self.assertTrue(isinstance(instance.path, CIMInstanceName))
-                self.assertTrue(len(instance.path.namespace) > 0)
+                self.assertTrue(instance.path.namespace, 'Includes namespace')
```

Comment:
The assertion message should say what the issue is when the assertion fails, not what it asserts.
I.e. change 'Includes namespace' to 'Namespace missing in instance path'.
Apart from that, is such a message really useful given that the test output shows the source line with the assertion?

Reply:
Fixed